sql/migrations/tracked_users/000001_init_schema.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE tracked_users_reports (
+                           id BIGSERIAL PRIMARY KEY,
+                           workspace_id VARCHAR(64) NOT NULL,
+                           instance_id VARCHAR(64) NOT NULL,
+                           source_id VARCHAR(64),
+                           reported_at TIMESTAMPTZ NOT NULL,
+                           userid_hll TEXT NOT NULL,
+                           anonymousid_hll TEXT NOT NULL,
+                           identified_anonymousid_hll TEXT NOT NULL
+);
+
+CREATE INDEX idx_tracked_users_reports_reported_at ON tracked_users_reports (reported_at);
```

Comment:
Is this the right index for our query pattern? Do we need source_id included as well?

Well we can always change this so not a blocker

Reply:
As per my understanding we would query everything and aggregate in memory. Correct me if I am wrong here @satishrudderstack